x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/importing.md

```diff
@@ -0,0 +1,72 @@
+Feature: Importing Prebuilt Rules
+
+  Scenario: Importing an unmodified prebuilt rule with a matching rule_id and version
+    Given the import payload contains a prebuilt rule with a matching rule_id and version, identical to the published rule
+    When the user imports the rule
+    Then the rule should be created or updated
+    And the ruleSource type should be "external"
+    And isCustomized should be false
+
+  Scenario: Importing a customized prebuilt rule with a matching rule_id and version
+    Given the import payload contains a prebuilt rule with a matching rule_id and version, modified from the published version
+    When the user imports the rule
+    Then the rule should be created or updated
+    And the ruleSource type should be "external"
+    And isCustomized should be true
+
+  Scenario: Importing a prebuilt rule with a matching rule_id but no matching version
+    Given the import payload contains a prebuilt rule with a matching rule_id but no matching version
+    When the user imports the rule
+    Then the rule should be created or updated
+    And the ruleSource type should be "external"
+    And isCustomized should be true
```

Comment:
@rylnd the 2 scenarios above are independent of the overwrite flag?
For both cases, the import should be successful even with flag overwrite=false?